net-vpn/headscale/headscale-0.15.0.ebuild

```
@@ -0,0 +1,39 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module systemd
+
+DESCRIPTION="An open source, self-hosted implementation of the Tailscale control server"
+HOMEPAGE="https://github.com/juanfont/headscale"
+SRC_URI="
+	https://github.com/juanfont/headscale/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~jsmolic/distfiles/net-vpn/headscale/${P}-deps.tar.xz
+"
+LICENSE="BSD Apache-2.0 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	acct-group/headscale
+	acct-user/headscale
+"
+
+RDEPEND="
+	${DEPEND}
+	net-firewall/iptables
+"
+
+src_install() {
+	dobin headscale
+	keepdir /var/lib/headscale
+	keepdir /etc/headscale
```

You can add some newlines in src_install to make it more readable if you wish
```suggestion
	keepdir /etc/headscale
	
```

---

maybe also move the first line dobin (L 29) down behind keepdir (L31)